src/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -6,6 +6,8 @@
 // ------------------------------------------------------------------------------
 
 
+using System.IO;
```

Comment:
The automated tool doesn't use usings, so we avoid doing so when manually changing the ref files, as it'll then churn the next time we run the tool.  Please just change the FileStream references you've added to be System.IO.FileStream.

Reply:
I don't understand this comment fairly accurate due to my knowledge of English. I should remove this using?

Reply:
Yes please remove the using. In ref files we use fully qualified type names.

Reply:
I removed using and used fully qualified type names.